io/GDALWriter.cpp

```diff
@@ -321,6 +329,45 @@ void GDALWriter::doneFile()
         throwError(raster.errorMsg());
 
     getMetadata().addList("filename", m_filename);
+
+    std::vector<std::string> gdalitems = Utils::split(m_GDAL_metadata, ',');
+    for (auto& v: gdalitems)
+    {
+        const std::size_t pos = v.find_first_of("=");
```

Comment:
I'm not sure why we'd accept '=' in a value. Does GDAL? Seems error-prone.

Reply:
someone giving us base64 to put into GDAL's metadata is going to be giving us '='.